index.html

```diff
@@ -128,5 +113,184 @@ <h1 class="primary-headline">
         </div>
       </div>
     </section>
+    <section id="work">
+      <div class="wrapper">
+        <div class="work-container">
+          <h2 class="work-title section-title">My Recent Works</h2>
+          <div class="cards-container">
+            <div class="card">
+              <div class="card-image"></div>
+              <div class="card-content">
+                <div class="card-title">Multi-Post Stories Gain+Glory</div>
+                <ul class="card-items">
+                  <li class="card-item">Ruby on Rails</li>
+                  <li class="card-item">CSS</li>
+                  <li class="card-item">JavaScript</li>
+                  <li class="card-item">HTML</li>
+                </ul>
```

Comment:
You did a great job working on this section, but there is a litle thing you should work on to go to the next steps. 😊 
- Kindly I suggest you stick to the design as much as possible, including margins (top, bottom, left, right) as defined in the design.

> ## The Figma Design
>
><img width="349" alt="Screenshot 2023-06-07 at 3 50 49 PM" src="https://github.com/Titus-Kiplagat/Microverse-Portfolio-Project/assets/91237525/7522523b-afa4-4393-b896-cf6b6f624e8d">
>
> ## Your Design
>
><img width="373" alt="Screenshot 2023-06-07 at 3 51 49 PM" src="https://github.com/Titus-Kiplagat/Microverse-Portfolio-Project/assets/91237525/8d482b71-2a55-4f2a-bf89-890f88b4720c">

Reply:
Hi **@stanleySimeon** 

Thank you for your insightful feedback on my code review. I appreciate your attention to detail and the time you've taken to provide suggestions. 
- I want to let you know that the first issue that's related to the README file is in the main branch after merging the first PR.
- Unfortunately, I'm unable to provide a live demo link at this time because the project has not yet been hosted.

I truly value your expertise and perspective. If you have any further ideas or insights on how to enhance the project, I would be more than open to hearing them. Your input is highly valuable to me, and I believe it will contribute to making the project even better.